deep_qa/training/trainer.py

```diff
@@ -319,7 +319,7 @@ def train(self):
         # so you may have left it above zero on accident.
         if self.validation_arrays is not None:
             kwargs['validation_data'] = self.validation_arrays
-        elif self.validation_split > 0.0:
+        elif self.validation_split > 0.0 and isinstance(self.training_arrays, tuple):
```

Comment:
Can't it be in `Trainer` with a default value of false? That way you'd have to pass it explicitly in the parameter file, and presumably you'd know whether or not your subclass can handle it. But yeah, a field would also be fine.